src/SFML/Window/Win32/JoystickImpl.cpp

```diff
@@ -473,13 +474,9 @@ void JoystickImpl::cleanupDInput()
 bool JoystickImpl::isConnectedDInput(unsigned int index)
 {
     // Check if a joystick with the given index is in the connected list
-    for (const JoystickRecord& record : joystickList)
-    {
-        if (record.index == index)
-            return true;
-    }
-
-    return false;
+    return std::any_of(joystickList.cbegin(),
+                       joystickList.cend(),
+                       [index](const JoystickRecord& record) { return record.index == index; });
```

Comment:
`std::find()`?

Possibly also above...

Reply:
`std::any_of` has the virtue of returning `bool` directly

Reply:
I see, OK.


<details><summary>Off-topic:</summary>

To be fair, either way is not great, STL algorithms still lack expressive power in 2023. 
The Kotlin equivalent of these 3 lines:
```kotlin
return joystickList.any { it.index == index }
```

</details>